doc/programming_model/data_types.md

```diff
@@ -26,10 +25,6 @@ operations which take bool as inputs and/or outputs data type.
 | f4\_e3m0  | 4-bit floating-point with 3 exponent bits and no mantissa bit                                                                                                                           |
 
 
-@note
-    Boolean is only supported in the Graph Compiler in CPU engines. No
-    primitives support boolean during primitive computation.
```

Comment:
If `boolean` is not supported by any other functionality shall we drop it from the table above as well?

+@uxlfoundation/onednn-arch

Reply:
It's supported by graph API to define the causal masks.